packages/nodes-base/nodes/Webhook/Webhook.node.ts

```diff
@@ -45,7 +45,8 @@ export class Webhook extends Node {
 		icon: { light: 'file:webhook.svg', dark: 'file:webhook.dark.svg' },
 		name: 'webhook',
 		group: ['trigger'],
-		version: [1, 1.1, 2],
+		version: [1, 1.1, 2, 2.1],
+		defaultVersion: 2,
```

Comment:
if this is meant as a temprory feature flag for the feature, would be good to leave a comment. Otherwise another developer might add a new default version and streaming responses would be rolled out.